mappings/net/minecraft/block/PointedDripstoneBlock.mapping

```diff
@@ -1,15 +1,34 @@
 CLASS net/minecraft/unmapped/C_mqztguph net/minecraft/block/PointedDripstoneBlock
 	FIELD f_cizsryow STALACTITE_FLOOR_SEARCH_RANGE I
 	FIELD f_cttqfcoe BASE_SHAPE Lnet/minecraft/unmapped/C_zscvhwbd;
+	FIELD f_elvssjrw MAX_SEARCH_LENGTH_BETWEEN_STALACTITE_TIP_AND_CAULDRON I
+	FIELD f_fafmlqec MAX_SEARCH_LENGTH_WHEN_CHECKING_DRIP_TYPE I
+	FIELD f_fbahmvfg STALACTITE_DRIP_START_PIXEL F
+	FIELD f_geyyeqmy DELAY_BEFORE_FALLING I
+	FIELD f_gllsmtmr MAX_HORIZONTAL_OFFSET F
 	FIELD f_gxticsyy TIP_MERGE_SHAPE Lnet/minecraft/unmapped/C_zscvhwbd;
 	FIELD f_igqmwyaf WATERLOGGED Lnet/minecraft/unmapped/C_xhwijdsd;
 	FIELD f_ijopvefq FRUSTUM_SHAPE Lnet/minecraft/unmapped/C_zscvhwbd;
 	FIELD f_ipkmpdju THICKNESS Lnet/minecraft/unmapped/C_cgckxfsw;
+	FIELD f_jddhgoav LAVA_CAULDRON_FILL_CHANCE_PER_RANDOM_TICK F
+	FIELD f_jgybmyto STALACTITE_DAMAGE_PER_FALL_DISTANCE_AND_SIZE F
+	FIELD f_jwzgoxfc STALAGMITE_FALL_DISTANCE_OFFSET F
+	FIELD f_kdusyxft MAX_STALACTITE_HEIGHT_FOR_DAMAGE_CALCULATION I
+	FIELD f_kgtqoixj STALAGMITE_FALL_DAMAGE_MODIFIER I
 	FIELD f_knqnihnn MIDDLE_SHAPE Lnet/minecraft/unmapped/C_zscvhwbd;
+	FIELD f_lerhamqv MAX_SEARCH_LENGTH_WHEN_LOOKING_FOR_TIP_OF_FALLING_STALACTITE I
```

Comment:
Very cursed lol. MAX_SEARCH_LENGTH_FOR_TIP_OF_STALACTITE But like is it just length? Im a bit confused with this one because how is it specifically looking for the tip too?

Reply:
I'm not overly sure, i can't find any usages of it in the code, it seems to just be an unused field? or i'm dumb

Reply:
It was removed in 22w05a